addons/account-settings-capitalize/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Capitalized Account Settings",
+  "name": "Capitalize Account Settings",
```

Comment:
I don't think this needs to be changed as it was already a noun.

Reply:
I think I added a single comment instead of starting a review, wait for more comments...

Reply:
It's also an Easter egg so i don't necessarily think it's worth changing either.

Reply:
It still makes the same amount of sense, and it's shorter.

The question is, are we okay with it being a phase like it now is?

Reply:
What about
```suggestion
  "name": "Account Settings Capitalization",
```
It is longer though

Reply:
I think "Account Settings" needs to be in quotation marks to make this clearer.